### PR TITLE
Designcourse

### LINKED ISSUE #351: DesignCourse
### URL

https://www.youtube.com/@DesignCourse/featured

### Description of the above provided site

This channel is great for learning css and frontend.
It will give you paid css course 

### Category

Frontend

### Sub Category

cat = youtube
subcat = css

### PR DESCRIPTION
**Pull request by parma1111**
## Fixes Issue

  Closes #351  

## Changes proposed

design course added

## Screenshots
Screenshot (768)
Screenshot (770)

## Note to reviewers

Give me progressive feedback

### CODE REVIEW

File: database/youtube/css.json [hunk 1]
```diff
@@ -14,5 +14,13 @@
         "category": "youtube",
         "subcategory": "css",
         "language": "english"
+    },
+    {
+        "name": "DesignCourse",
+        "description": "This channel is great for learning css and frontend. It will give you paid course for css.",
```
Comment (CBID2) [inline database/youtube/css.json:20]:
```suggestion
        "description": "This channel is great for learning css and frontend.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @parma1111! I made a suggestion to your PR.

File: database/youtube/css.json (hunk 1 above)
Comment (parma1111) [reply database/youtube/css.json]: I have to remove red line description CBID2 ?

**Review by CBID2 — APPROVED** [review]
LGTM! 😊👍🏾

**Comment by parma1111** [comment]
> thank you Parma!

It's my pleasure 😀

**Pull request merged**